### DISCUSSION TITLE
How do I use V2 FS Plugin?

### DISCUSSION THREAD

**Discussion by humblemodulo** [tauri-apps/tauri#10726] [Q&A]
So, I'm fully willing to accept that the issue is EBCAK, but I need an assist here.

Trying to make use of the FS plugin. I'm following the latest 2.0 docs under "Migrate to File System Plugin":

- Cargo.toml -> `tauri-plugin-fs = "2"`
  - ERROR: complains that the latest version is `2.0.0-rc.0`

- package.json -> `"@tauri-apps/plugin-fs": "^2.0.0"`
  - ERROR: complains that the latest version is `2.0.0-rc.1`

- Main.rs -> `.plugin(tauri_plugin_fs::init())`
  - ERROR: complains `PluginInitialization("fs", "Error deserializing 'plugins.fs' within your Tauri configuration: unknown field `all`, expected `requireLiteralLeadingDot`")`

I make the relevant adjustments to Cargo.toml and package.json as far as I can tell, but the `PluginInitialization`  issue remains.  Can someone point me in the direction of up-to-the-second documentation?  😂

**Comment by humblemodulo** [comment]
Hrm, no idea why but it seems that removing `scopes` from `tauri.conf.json` "resolved" the remaining issue.  🤔

**Comment by thiagobessimo** [comment]
I am having the same issue. It shows:

```
thread 'main' panicked at src/main.rs:10:10:
error while running tauri application: PluginInitialization("fs", "Error deserializing 'plugins.fs' within your Tauri configuration: unknown field `scope`, expected `requireLiteralLeadingDot`")
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I am really not sure how to even approach finding a solution since documentation only mentions adding a scope field

**Reply by FabianLars** [reply]
You're in the wrong file. Scopes belong into the capabilities files in v2 (where you also set permissions): https://v2.tauri.app/plugin/file-system/#examples

**Reply by thiagobessimo** [reply]
Thanks! I'll check it out.

**Reply by thiagobessimo** [reply]
Just FYI, I was following this page in the documentation

https://v2.tauri.app/reference/javascript/fs/#security

**Reply by FabianLars** [reply]
oh damn, thanks for telling me

**Reply by MeepTech** [reply]
I also came here from that page after trying for almost a day using those examples. Should probably update the docs.
